### PR TITLE
Added skip video option for authors

### PR DESCRIPTION
**Pull request by narendrasss**
Implements #145

### CODE REVIEW

**Comment by narendrasss** [comment]
@wynnset I'm not entirely sure with my solution to adding the close button when the video progress is over 95%. It works, but I don't like that I'm using event dispatchers. Let me know if you can think of something better.

**Comment by narendrasss** [comment]
@wynnset Should be fixed.

File: templates/tapestry.js [hunk 1]
```diff
@@ -130,6 +131,7 @@ function tapestryTool(config){
             if (config.wpUserId) { // Get from database if user is logged in
 
                 jQuery.get(USER_NODE_PROGRESS_URL, { "post_id": config.wpPostId }, function(retrievedUserProgress) {
+                    console.log(retrievedUserProgress);
```
Comment (wynnset) [inline templates/tapestry.js]: Please do not leave development console logs in ready PRs.

File: templates/tapestry.js [hunk 2]
```diff
@@ -2216,6 +2264,20 @@ function tapestryTool(config){
             console.error(e);
         });
     }
+
+    function saveNodeAsSkippable(node) {
+        console.log(node);
```
Comment (wynnset) [inline templates/tapestry.js]: Please do not leave console logs in ready PRs.

File: templates/tapestry.js [hunk 3]
```diff
@@ -1489,6 +1528,13 @@ function tapestryTool(config){
                     }
                     updateViewedValue(id, video.currentTime, video.duration);
                     updateViewedProgress();
+                    if (video.currentTime / video.duration >= 0.95) {
```
Comment (wynnset) [inline templates/tapestry.js]: Please make 0.95 into a class attribute or configurable variable at the top.

File: templates/tapestry.js [hunk 4]
```diff
@@ -37,6 +37,7 @@ function tapestryTool(config){
         API_DELETE_METHOD = 'DELETE',
         USER_NODE_PROGRESS_URL = config.apiUrl + "/users/progress",
         USER_NODE_UNLOCKED_URL = config.apiUrl + "/users/unlocked",
+        USER_NODE_SKIP_URL = config.apiUrl + "/users/skip",
```
Comment (wynnset) [inline templates/tapestry.js]: This should probably be named `skipped` to be consistent (everything else is a noun, not a verb)

**Pull request merged**